pandas/tests/test_index.py

```diff
@@ -297,10 +297,6 @@ def test_constructor(self):
         # arr = np.array(5.)
         # self.assertRaises(Exception, arr.view, Index)
 
-    def test_constructor_corner(self):
-        # corner case
```

Comment:
It has to do with the extension of the `Index()` constructor to allow it be used as a short-cut to constructing a `RangeIndex`. This segment tested that e.g. `Index(0)` raised an error since only arrays were valid arguments.

My proposal was that e.g. `Index(5)` could return a `RangeIndex`. The previously invalid constructor use would thus now be valid.